project_report/tex/theory_and_methods.tex

```diff
@@ -1,5 +1,35 @@
 \section{Theory \& Methods}
 \subsection{Theory}
+\subsubsection{Automation Pyramid}
+The automation pyramid is a graphical representation of the different levels of
+automation in a factory or industry, as seen in Figure \ref{figure:ap}.
+
+\begin{figure}[ht]
+	\centering 
+	\includegraphics[width=1\linewidth]{images/automation_pyramid}
+	\caption{Illustration of the Automation Pyramid}
+	\label{figure:ap}
+\end{figure}
+
+
+The first level, the field level, consists of the devices, actuators, and
+sensors that are used on the production floor. The control level is where PLC's
+(Programmable Logic Controller) and PID's (Proportional-Integral-Derivative
+controller) are located. These are used to control the devices in the field
+level. On the third layer, the supervisory level, which utilises SCADA. SCADA
```

Comment:
Remove On and which.

"The third layer, the supervisory level, utilises SCADA."